### ISSUE TITLE
Usage doc bug

### ISSUE THREAD

**Issue by hartwork** [nir0s/distro#171]
The docs read
```
$ python
>>> import distro
>>> distro.linux_distribution(full_distribution_name=False)
'('centos', '7.1.1503', 'Core')'
```
but the last line seems to have too many quotes: The left- and rightmost ones.
Please compare with my local output:
```
# python3.5
[..]
>>> import distro
>>> distro.linux_distribution(full_distribution_name=False)
('gentoo', '2.3', 'n/a')
```

**Comment by nir0s** [comment]
Thanks, will fix that.

**Comment by nir0s** [comment]
Fixed with 1.0.3
